app/scripts/lib/app-start.js

```diff
@@ -257,21 +257,25 @@ define(function (require, exports, module) {
         let relier;
         const context = this._getContext();
 
-        if (this._isServiceSync()) {
-          // Use the SyncRelier for sync verification so that
-          // the service name is translated correctly.
-          relier = new SyncRelier({ context }, {
+        // The check for whether the user is in an OAuth flow
+        // is done first to ensure OAuth reliers that specify
+        // service=sync in addition to client_id still get the
```

Comment:
`OAuth reliers that specify service=sync`, that should never be the case, so I'm a bit confused by this comment